samples/benchmarks/coremark/README.rst

```diff
@@ -0,0 +1,146 @@
+.. _coremark_sample:
+
+CoreMark
+########
+
+.. contents::
+   :local:
+   :depth: 2
+
+The sample demonstrates how to run the `CoreMark®`_ benchmark to evaluate performance of a core.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+
+The sample runs the CoreMark benchmark on the target CPU.
+CoreMark evaluates the CPU efficiency by performing different algorithms, such as state machine, CRC calculation, matrix manipulation, and list processing (find and sort).
+
+To run the CoreMark benchmark on the preferred core, press the corresponding button.
+For the button assignment, see the :ref:`coremark_user_interface` section.
+When the benchmark finishes running, you can restart it by pressing the same button.
+If you want to run the sample upon the startup, enable the :kconfig:option:`COREMARK_MODE_FLASH_AND_RUN` Kconfig option.
+
+.. _coremark_user_interface:
+
+User interface
+**************
+
+Each target CPU has an assigned button responsible for starting the benchmark and LED that indicates the ``test in progress`` state:
+
+Button 1:
+   Start the benchmark run on the application core.
+
+Button 2:
+   Start the benchmark run on the network/radio core.
```

Comment:
```suggestion
   Start the benchmark run on the network or radio core (or both).
```
If it is just one core, then `network (radio) core`.

Reply:
same as above

Reply:
As mentioned by Pekka, we don't really use `/` in docs to indicate alternatives, as they could mean `X or Y` or `X and Y`.